change/@azure-communication-react-1e34bbb0-f4f9-4e61-83e9-09f34513f6ca.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
not sure why is there 4 changelog files, i think you just need one for `change` and one for `change-beta`